src/app/survey/survey.component.html

```diff
@@ -0,0 +1,29 @@
+<h3> Teacher Survey </h3>
```

Comment:
merge conflicts in this file, but this might should be completely removed anyway.  Make the changes here (if any) to `app/review/review.component.html`